### ISSUE TITLE
Pylance crash: Debug Failure. Unable to find notebook URI for vscode-notebook-cell:

### ISSUE THREAD

**Issue by MySlientWind** [microsoft/pylance-release#5037]
## Environment data

-   Language Server version: Pylance language server 2023.10.50 (pyright cfe56bfb)
-   OS and version: Linux version 4.18.0-477.13.1.el8_8.x86_64 (mockbuild@iad1-prod-build001.bld.equ.rockylinux.org) (gcc version 8.5.0 20210514 (Red Hat 8.5.0-18) (GCC)) `#1` SMP Tue May 30 22:15:39 UTC 2023
-   Python version (& distribution if applicable, e.g. Anaconda): 3.9.18

## Code Snippet

Do not need code

## Repro Steps

Open a notebook file in remote server

## Expected behavior

Pylance works well

## Actual behavior
 
Pylance can not work, and raise the error in output: python language server

## Logs

```
2023-10-28 15:35:24.565 [info] Error: Debug Failure. Unable to find notebook URI for vscode-notebook-cell://ssh-remote%2B7b22686f73744e616d65223a224d65746973227d/home/{my_user_name}/path/to/my/notebook.ipynb#W1sdnNjb2RlLXJlbW90ZQ%3D%3D
    at _0x4f6f08._getNotebookUri (/home/{my_user_name}/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:377799)
    at _0x4f6f08.getNotebookPathFromCell (/home/{my_user_name}/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:375944)
    at _0x4f6f08.getNotebookCellPath (/home/{my_user_name}/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:375163)
    at _0x29682b.decodeTextDocumentUri (/home/{my_user_name}/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:347323)
    at /home/{my_user_name}/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:1086078
    at Array.map (<anonymous>)
    at _0x4ec48a.<computed> (/home/{my_user_name}/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:1086017)
    at _0x5e13d0._onDidOpenNotebookDocumentAsync (/home/{my_user_name}/.vscode-server/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:1066530)

```

**Comment by MySlientWind** [comment]
Pylance works well for `.py` file

**Comment by SINNED0221** [comment]
我也有类似的问题，但是是本地的py文件
总之回滚到上个版本还能用

**Comment by MySlientWind** [comment]
> 我也有类似的问题，但是是本地的py文件 总之回滚到上个版本还能用

多谢，我这边回到 2023.10.21 (pyright 088ebaa5) 就好了

There is no such problem in version 2023.10.21 (pyright 088ebaa5)

**Comment by mostirontwig123** [comment]
been having the same problem i checked fire wall deleted reinstalled then unistalled vs and reinstalled lmk if you fixed it i havent been able to figure out why

**Comment by mostirontwig123** [comment]
side note i got it to work up until i run the code then i just seems to break crash restart then i get the error again so im clueless nw

**Comment by fdx24** [comment]
> 我也有类似的问题，但是是本地的py文件 总之回滚到上个版本还能用
> 
> Translated: I have a similar issue, but it's a local py file Anyway, rolling back to the previous version still works

谢谢，终于搞定了

**Comment by mostirontwig123** [comment]
is there a fix then or do i just wait for vs to fix it ?

**Comment by Symbyos** [comment]
I had the exact same issue. I have downgraded Pylance to `v2023.10.40` and now it works fine.

**Comment by mostirontwig123** [comment]
> I had the exact same issue. I have downgraded Pylance to `v2023.10.40` and now it works fine.

Awesome thanks for the helped just switched and it started working !!

**Comment by marcosandregoncalves** [comment]
> I had the exact same issue. I have downgraded Pylance to `v2023.10.40` and now it works fine.

How do you downgraded?

**Comment by marcosandregoncalves** [comment]
done it. Thanks :D

**Comment by Looby72** [comment]
> I had the exact same issue. I have downgraded Pylance to `v2023.10.40` and now it works fine.

Also worked for me. Thanks :))

You can install an earlier version using dropdown menu of install/uninstall button on the plugin page inside VS Code.

**Comment by albertocastany** [comment]
> I had the exact same issue. I have downgraded Pylance to `v2023.10.40` and now it works fine.

I can confirm this. I've downgraded to `v2023.10.40`, and I haven't experienced any crash while working with Jupyter Notebooks.

**Comment by MPruszewicz** [comment]
I also can confirm,
downgraded to v2023.10.40

**Comment by jan-ninja** [comment]
similar issue here with devcontainer, no crashes here since downgrading to `v2023.10.40`

**Comment by BillJWatts** [comment]
Upgrading to the prerelease version `v2023.10.53` also fixed the issue for me

**Comment by rchiodo** [comment]
Thanks this issue should be fixed in 2023.10.53
